frontend/src/lib/components/LemonBubble/LemonBubble.tsx

```diff
@@ -0,0 +1,38 @@
+import clsx from 'clsx'
+import React from 'react'
+import { CSSTransition } from 'react-transition-group'
+import './LemonBubble.scss'
+
+export interface LemonBubbleProps {
+    count?: number
+    size?: 'small' | 'medium' | 'large'
+    position?: 'none' | 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right'
+    showZero?: boolean
+}
+
+/** An icon-sized Bubble for displaying a count.
+ *
+ *  Numbers up to 9 are displayed in full, in integer form, with 9+ for higher values
+ */
+export function LemonBubble({
+    count,
+    size = 'medium',
+    position = 'none',
+    showZero = false,
+}: LemonBubbleProps): JSX.Element {
+    console.log(showZero)
+    // NOTE: We use 1 for the text if not showing so the fade out animation looks right
+    const text = typeof count === 'number' && count !== 0 ? (count < 10 ? String(count) : '9+') : showZero ? '0' : '1'
+    const hide = count === undefined || (count == 0 && !showZero)
+
+    return (
+        <CSSTransition in={!hide} timeout={250} classNames="anim-" mountOnEnter unmountOnExit>
```

Comment:
Total nits:
I've usually been using just the component name (i.e.  `classNames="LemonBubble-"`) if the `CSSTransition` applies to the whole component appearing/disappearing (then you get obvious classes like `.LemonBubble--enter-active`).
As a microinteraction 250 ms feels a _bit_ slow, maybe 200?